csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -542,7 +542,7 @@ object CsvValidatorUi extends SimpleSwingApplication {
       fileButton.reactions += {
         case ev: ButtonClicked =>
           val startingDir = if(toPathField.text.isEmpty) userDir.toFile else Path.of(toPathField.text).toFile
-          val fromFolderName = Path.of(fromPathField.text).getFileName
+          val fromFolderName = Path.of(URI.create(fromPathField.text)).getFileName
```

Comment:
This will work only when the path is a valid URI (i.e. it is `file://<whatever-path-to-file`), if it is just a path (i.e. `c:\<whatever-path-to-file`) we will get URI exception. Can we be sure that it is always URI? Otherwise it may be good idea to have a fallback plan of the existing line of code as well.

Reply:
This was Sam's fix, so I'll ask him. @MancunianSam, what do you think?